cmd/gin-repod/repo.go

```diff
@@ -54,10 +54,17 @@ func (s *Server) createRepo(w http.ResponseWriter, r *http.Request) {
 
 	rid := store.RepoId{Owner: owner, Name: creat.Name}
 
-	_, ok := s.checkAccess(w, r, rid, store.NoAccess)
+	user, ok := s.checkAccess(w, r, rid, store.NoAccess)
 	if !ok {
 		return
 	}
+	// make sure routes user and token user are identical
+	if owner != user.Uid {
+		http.Error(w, "Invalid repository owner name", http.StatusBadRequest)
+		fmt.Fprintf(os.Stderr,
```

Comment:
I'll leave this for now until we get the chance to make it consistent throughout the whole project.